utils/relay-manager.js

```diff
@@ -67,10 +67,13 @@ class RelayManager {
       + 4 * empty calldata bytes
       + 70k buffer
     */
-    let gas = gasLimit + 21000 + nonZeros * 16 + zeros * 4 + 70000;
+    let gas = gasLimit + 21_000 + nonZeros * 16 + zeros * 4 + 70_000;
```

Comment:
Wow, I didn't know javascript supported numeric separators :-)

Reply:
It gets tricky when there are lots of 0's :)